wavenet.py

```diff
@@ -210,7 +210,7 @@ def predict_proba(self, waveform, name='wavenet'):
                             [1, self.quantization_channels])
             return tf.reshape(last, [-1])
 
-    def loss(self, input_batch, name='wavenet'):
+    def loss(self, input_batch, C=0., name='wavenet'):
```

Comment:
The `C` parameter could use a more descriptive name, like `regularization_strength`, `regularization_coefficient`, or just `regularization`.
We should also set it to `None` by default, and not create any of the regularization ops below if it's `None`.
That way we the user doesn't anything to the TensorFlow graph when they don't want to use regularization.